_includes/manuals/nightly/3.2.2_installer_options.md

```diff
@@ -1681,35 +1681,35 @@ kafo-export-params -c /etc/foreman-installer/scenarios.d/foreman.yaml -f html
       </tr>
       <tr>
         <td style='white-space:nowrap'>--puppet-server-group</td>
-        <td>Name of the puppetmaster group.</td>
+        <td>Name of the puppet server group.</td>
```

Comment:
You are updating a generated document so it'll be updated on the next sync. https://github.com/theforeman/puppet-puppet is the right place to submit these.